### PR TITLE
refine some errors' message

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
Some error messages are not easy to understand and needs to be improved.

### What is changed and how it works?
Refine part of error messages.

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: pkg/column-mapping/column.go [hunk 1]
```diff
@@ -335,13 +335,13 @@ func (m *Mapping) queryColumnInfo(schema, table string, columns []string) (*mapp
 	var rule *Rule
 	if len(table) == 0 || len(tableRules) == 0 {
 		if len(schemaRules) != 1 {
-			return nil, errors.NotSupportedf("route %s/%s to rule set(%d)", schema, table, len(schemaRules))
+			return nil, errors.NotSupportedf("`%s`.`%s` matches %d route rules which is more than one. It's", schema, table, len(schemaRules))
```
Comment (csuzhangxc) [inline pkg/column-mapping/column.go]: Is `route` accurate here for `column mapping`?

File: pkg/table-router/router.go [hunk 2]
```diff
@@ -155,15 +155,15 @@ func (r *Table) Route(schema, table string) (string, string, error) {
 	)
 	if len(table) == 0 || len(tableRules) == 0 {
 		if len(schemaRules) > 1 {
-			return "", "", errors.NotSupportedf("route %s/%s to rule set(%d)", schema, table, len(schemaRules))
+			return "", "", errors.NotSupportedf("`%s`.`%s` matches %d route rules which is more than one.\nThe first two rules are %+v, %+v.\nIt's", schemaRules[0], schemaRules[1], schema, table, len(schemaRules))
```
Comment (csuzhangxc) [inline pkg/table-router/router.go]: Is the order of arguments correct?

**Comment by lichunzhu** [comment]
Revised. @csuzhangxc

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Pull request merged**